cmd/api-errors.go

```diff
@@ -858,6 +859,11 @@ var errorCodes = errorCodeMap{
 		Description:    "Storage backend has reached its minimum free disk threshold. Please delete a few objects to proceed.",
 		HTTPStatusCode: http.StatusInsufficientStorage,
 	},
+	ErrParentIsObject: {
+		Code:           "XMinioParentIsObject",
+		Description:    "Parent is already an object, please choose a different object name.",
```

Comment:
I think `Object-prefix` is better than `Parent` in
```
"Parent is already an object, ...
```
and `..., please choose a different object name"` could be confusing for some.
I think it would be more clear if the error message says:
```
"Object-prefix is already an object, please choose a different object-prefix name."
```

Reply:
Agreed @ebozduman